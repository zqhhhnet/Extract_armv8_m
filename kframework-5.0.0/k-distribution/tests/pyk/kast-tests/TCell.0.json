{"format":"KAST","version":1,"term":{"node":"KApply","label":"<T>","variable":false,"arity":2,"args":[{"node":"KApply","label":"<k>","variable":false,"arity":1,"args":[{"node":"KApply","label":"int_;_","variable":false,"arity":2,"args":[{"node":"KApply","label":"_,_","variable":false,"arity":2,"args":[{"node":"KToken","sort":"Id","token":"x"},{"node":"KApply","label":"_,_","variable":false,"arity":2,"args":[{"node":"KToken","sort":"Id","token":"y"},{"node":"KApply","label":".List{\"_,_\"}","variable":false,"arity":0,"args":[]}]}]},{"node":"KApply","label":"{}","variable":false,"arity":0,"args":[]}]}]},{"node":"KApply","label":"<state>","variable":false,"arity":1,"args":[{"node":"KApply","label":".Map","variable":false,"arity":0,"args":[]}]}]}}
