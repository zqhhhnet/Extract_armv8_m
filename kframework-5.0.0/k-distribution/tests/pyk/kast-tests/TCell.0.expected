<T>
  <k>
    int x, y, .Ids;
    { }
  </k>
  <state>
    .Map
  </state>
</T>

